integration-tests/src/main/resources/junit-platform.properties

```diff
@@ -0,0 +1 @@
+junit.jupiter.testclass.order.default=org.junit.jupiter.api.ClassOrderer$ClassName
```

Comment:
FYI, I added a `@TestProfile` aware orderer to Quarkus in...hum...2.3? It's mentioned here: https://quarkus.io/guides/getting-started-testing#testing_different_profiles

Reply:
Awesome, a much better option, and should help as we add more test profiles.  Thank you!